app/experimenter/kinto/tasks/nimbus.py

```diff
@@ -0,0 +1,164 @@
+import markus
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from experimenter.celery import app
+from experimenter.experiments.api.v6.serializers import NimbusExperimentSerializer
+from experimenter.experiments.changelog_utils import generate_nimbus_changelog
+from experimenter.experiments.models import (
+    NimbusBucketRange,
+    NimbusExperiment,
+    NimbusIsolationGroup,
+)
+from experimenter.kinto.client import KintoClient
+
+logger = get_task_logger(__name__)
+metrics = markus.get_metrics("kinto.tasks")
+
+
+def get_kinto_user():
+    user, _ = get_user_model().objects.get_or_create(
+        email=settings.KINTO_DEFAULT_CHANGELOG_USER,
+        username=settings.KINTO_DEFAULT_CHANGELOG_USER,
+    )
+    return user
+
+
+@app.task
+@metrics.timer_decorator("push_experiment_to_kinto.timing")
+def nimbus_push_experiment_to_kinto(experiment_id):
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("push_experiment_to_kinto.started")
+
+    experiment = NimbusExperiment.objects.get(id=experiment_id)
```

Comment:
Does it make sense to double-check here that the status is suitable for being pushed to kinto? Or are we sufficiently confident that could never happen?

Reply:
Yeah that logic lives in the other task, this one should just push whatever it's given.